kolibri/plugins/learn/assets/src/views/LibraryPage/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+
+  
+  <KCard
+    to="to"
+    :headingLevel="headingLevel"
+    layout="horizontal"
+    thumbnailDisplay="large"
+    :title="contentNode.title"
+    :thumbnailSrc="thumbnailSrc"
+    :thumbnailScaleType="thumbnailScaleType"
+    :thumbnailAlign="thumbnailAlign"
+    :style="{ height: '172px', margin: '16px 0 16px 0' }"
+  >
+    <template #thumbnailPlaceholder>
+      <div class="default-folder-icon" >
+        <KIcon
+          icon="topic"
+          :color="$themePalette.grey.v_600"
+          class=""
+        />
+      </div>
+    </template>
+
+    <template #belowTitle>
+      <div
+        class="header-bar"
+        :style="headerStyles "
+      >
+        <KIcon
+          icon="topic"
+          :color="$themePalette.grey.v_800"
+          class="folder-header-bar"
+        />
+        <p
+          class="folder-header-text"
+          :style="{ color: $themePalette.grey.v_600 }"
+        >
+          {{ coreString('folder') }}
+        </p>
+      </div>
+    </template>
+
+  </KCard>
+
+</template>
+
+
+<script>
+
+  import { validateLinkObject } from 'kolibri.utils.validators';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import commonLearnStrings from '../commonLearnStrings';
+
+  export default {
+    name: 'AccessibleFolderCard',
+    mixins: [commonLearnStrings, commonCoreStrings],
+    props: {
+      to: {
+        type: Object,
+        required: true,
+        validator: validateLinkObject,
+      },
+      contentNode: {
+        type: Object,
+        required: true,
+      },
+      headingLevel: {
+        type: Number,
+        required:true
+      },
+      thumbnailSrc: {
+        type: String,
+        default: null,
+      },
+      thumbnailScaleType: {
```

Comment:
I think `thumbnailScaleType` and `thumbnailAlign` will be the same for all folder cards so doesn't need to be configurable from outside. Would suggest we remove these two props and just configure it right on `KCard` similarly to how you configured `layout`. Same applies to resource and channel cards (if relevant, didn't preview their props yet). Generally let's just expose APIs from these `Accessible...` cards that we know they need to be dynamic to make their usage simple and consistent (if there's some new expectations in the future, we can expose based on actual need)